src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/player/ModuleFastUse.kt

```diff
@@ -77,7 +77,51 @@ object ModuleFastUse : Module("FastUse", Category.PLAYER) {
             }
         }
     }
+    // Anticheat: Grim
+    // Tested AC Version: 2.5.34
+    // Tested on: eu.loyisa.cn, anticheat-test.com
+    // Usable MC version: 1.17-1.20.4
+    // Q: Why this works?
+    // A: https://github.com/GrimAnticheat/Grim/blob/9660021d024a54634605fbcdf7ce1d631b442da1/src/main/java/ac/grim/grimac/checks/impl/movement/TimerCheck.java#L99
+    private object Grim : Choice("Grim2.5.34") {
+        override val parent: ChoiceConfigurable
+            get() = modes
+
+        val repeatable = repeatable {
+            if (!player.isUsingItem) {
+                return@repeatable
+            }
+            if (player.activeItem.isFood || player.activeItem.item is MilkBucketItem
+                || player.activeItem.item is PotionItem
+            ) {
+                if (player.isUsingItem) { 
```

Comment:
`if (player.isUsingItem)` seems to be redundant here because there is already an appropriate condition in the line 91. Therefore, it can be removed as it's always true, isn't it?

Reply:
Yes. Good catch, its really not needed. If this is removed I will approve this PR.